src/containers/ProjectContentsContainer.jsx

```diff
@@ -53,6 +59,9 @@ class ProjectContentsContainer extends Component {
           <ModalBody>
             <p>{this.state.translationText}</p>
             <input autoFocus placeholder="Translate some text" />
+            <DefaultButton onClick={this.handleSubmit}>
+              Submit
+            </DefaultButton>
           </ModalBody>
           <ModalFooter>
             <DefaultButton onClick={() => this.setState({modalOpen: false})}>
```

Comment:
I've pushed the updated lock file, after a fresh install. There might have been something in your lock file commit that killed the install. I saw errors about SHA hash mismatches.